composer.json

```diff
@@ -17,7 +17,8 @@
     "require": {
         "php": "^7.1",
         "squizlabs/php_codesniffer": "~3.0",
-        "dealerdirect/phpcodesniffer-composer-installer": "^0.4.2"
+        "dealerdirect/phpcodesniffer-composer-installer": "^0.4.2",
+        "slevomat/coding-standard": "3.3.*"
```

Comment:
Can we require `^4.0` instead? They have changed the composer project type, then `dealerdirect/phpcodesniffer-composer-installer` will be able to install it automatically for us (and we don't need to use file paths)

Reply:
Yep, I'll update it and the way how we reference sniff. There was no 4.0 release back when I created this PR. 👍